test/sparse/sparse.jl

```diff
@@ -1642,9 +1642,9 @@ end
 
 # Test temporary fix for issue #16548 in PR #16979. Brittle. Expect to remove with `\` revisions.
 # This is broken by the introduction of RowVector... see brittle comment above.
-#@testset "issue #16548" begin
-#    @test which(\, (SparseMatrixCSC, AbstractVecOrMat)).module == Base.SparseArrays
-#end
+@testset "issue #16548" begin
+    @test_broken which(\, (SparseMatrixCSC, AbstractVecOrMat)).module == Base.SparseArrays
```

Comment:
Hmm, OK. At first I thought the issue might have been it going through a `rowvector.jl` method first before going to the `Sparse` module, but in fact this may be a disambiguity method that I add to `Sparse`, so that should work.